components/monitoring/prometheus/base/servicemonitors/pipeline-service.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: pipeline-service
+  namespace: appstudio-workload-monitoring
+  labels:
+    app: pipeline-metrics-exporter
+    prometheus: appstudio-workload
+spec:
+  selector:
+    matchLabels:
+      app: pipeline-metrics-exporter
+  endpoints:
+    - path: /metrics
+      port: metrics
```

Comment:
this port matches the service exporter port right?

Reply:
Yes, [here](https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/metrics-exporter/service.yaml#L11) is the link for it